varcode/effect_ordering.py

```diff
@@ -52,10 +52,18 @@ def top_priority_transcript_effect(effects):
         elif priority == best_priority:
             best_effects.append(effect)
 
-    def effect_cds_len(effect):
-        """
-        Returns length of coding sequence of transcript associated with effect
-        """
-        return len(effect.transcript.coding_sequence)
-
-    return max(best_effects, key=effect_cds_len)
\ No newline at end of file
+    if any(effect.transcript.complete for effect in best_effects):
```

Comment:
I'm not sure if I understand the question, so I'm not sure if this clarifies the assumptions: (1) the impact on a coding transcript is more interesting than the impact on a non-coding transcript (2) lacking any other info, a longer CDS is more interesting than a shorter one, (3) non-coding transcripts don't have a CDS, so instead sort them by full transcript length.